lemminx-maven/src/main/java/org/eclipse/lemminx/maven/MavenHoverParticipant.java

```diff
@@ -169,16 +168,31 @@ private String collectPuginConfiguration(IPositionRequest request) {
 		}
 		return null;
 	}
-	
-	private Hover toHover(String description) {
-		Hover hover = new Hover();
-		hover.setContents(new MarkupContent("plaintext", description));
-		return hover;
-	}
 
 	@Override
 	public String onText(IHoverRequest request) throws Exception {
-		// TODO Auto-generated method stub
+		DOMNode tag = request.getNode();
+		String tagText = tag.getNodeValue();
+		if (tagText.startsWith("${" ) && tagText.endsWith("}")) {
+			return collectProperty(request);
+		}
+		
+		return null;
+	}
+
+	private String collectProperty(IPositionRequest request) {
+		DOMDocument doc = request.getXMLDocument();
+		MavenProject project = cache.getLastSuccessfulMavenProject(doc);
+		String desiredProperty = request.getNode().getNodeValue();
+		desiredProperty = desiredProperty.substring(2, desiredProperty.length() - 1);
+		if (project != null) {
+			for (Entry<Object, Object> prop : project.getModel().getProperties().entrySet()) {
+				String property = (String) prop.getKey();
+				if (property.equals(desiredProperty)) {
+					return (String) prop.getValue();
```

Comment:
As mentioned on another discussion, this could be an enriched string like
```
Property: key
Value: value
```
so it's less ambiguous later we can more easily add content.

Reply:
Sounds good.